docker/docker-compose.yml

```diff
@@ -31,7 +32,7 @@ x-common-vars:
 # Templates.
 # ------------------------------------------------------------------------------
 x-stacks-blockchain: &stacks-blockchain
-  image: blockstack/stacks-core:3.2.0.0.2@sha256:67c7574782c3b7f95a592d46d881d261524a4bc1c5cf4e66d54d39f330e4bf69
+  image: blockstack/stacks-core:release-3.3.0.0.0-rc2@sha256:1b895681b073bc791f89182e9f985cc46d30552049ba8e9f7f16a19aec9c6a86
```

Comment:
Should we also update `docker/mainnet/nodes/docker-compose.chains.yml`? Asking since this is a not the real release, but also we usually keep those two aligned

Reply:
Agree in keeping them in sync (or otherwise just making sure we have one version everywhere)